config/dockergento/nginx/conf/default.conf

```
@@ -41,7 +48,7 @@ server {
     root $MAGE_ROOT;
     location ~ ^/setup/index.php {
       fastcgi_split_path_info ^(.+\.php)(/.+)$;
-      fastcgi_pass   fastcgi_backend;
+      fastcgi_pass   $my_fastcgi_pass:9001;
```

why do we need 9001 port here? As it's not php-fpm node -it's not conflicting anyhow with default 9000 port of xdebug.

Could you replace it to 9000?
```suggestion
      fastcgi_pass   $my_fastcgi_pass:9000;
```

Related to https://github.com/ModestCoders/magento2-dockergento/issues/49